### DISCUSSION TITLE
using ]f to go to next function doesn't work for .go file?

### DISCUSSION THREAD

**Discussion by juniway** [helix-editor/helix#2178] [Q&A] [answered]
When I press `]`, it pops up this menu, 
<img width="298" alt="CleanShot 2022-04-19 at 23 57 34@2x" src="https://user-images.githubusercontent.com/3980841/164045729-cad1b65e-4665-4388-b595-286d18055db0.png">

but when I then press `f`, nothing happens. Also for other keys like `o` or `c`, they don't work. Only `p` works.

**Comment by the-mikedavis** [comment, marked as answer]
This was just fixed yesterday: https://github.com/helix-editor/helix/issues/1920

Are you using 22.03 or building from source? You'll need to build from the latest source to see the fix.

**Reply by juniway** [reply]
Just now, I rebuilt hx with the latest commit on master branch, its version is now `helix 22.05-dev (cc68fa85)`, and my OS is macOS 12.3 Monterey, but looks like these jump keys still won't work.

**Reply by the-mikedavis** [reply]
Depending on how you built helix you may need to re-link the runtime directory. You may have stale queries for go there. https://github.com/helix-editor/helix/blob/cc68fa857dced7d9433d145ebfa8cd70e5a0394d/README.md?plain=1#L48-L50

**Reply by juniway** [reply]
thanks, it works after I removed old runtime and replaced it with the new one.
